Backend/.env.example

```
@@ -0,0 +1,21 @@
+JWT_EXPIRE=7d
+
+ML_URI=http://localhost:8000
+
+GNEWS_API=your_gnews_api_key
+
+CLOUD_NAME=your_cloudinary_cloud_name
+CLOUD_API_KEY=your_cloudinary_api_key
+CLOUD_SECRET=your_cloudinary_cloud_secret
+
+EMAIL_USER=your_gmail_address
+EMAIL_PASS=your_gmail_app_password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMAIL_PASS key should go before the EMAIL_USER key

(UnorderedKey)